tests/Feature/Http/Controllers/Auth/AuthenticationTest.php

```diff
@@ -76,7 +76,7 @@ public function testUsersCanNotAuthenticateWithInvalidPassword(): void
     public function testUsersCanLogoutFromTheirSession(): void
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````